history/warren-weaver.md

```diff
@@ -0,0 +1,11 @@
+---
+description: One of the pioneers of machine translation
+---
+# Warren Weaver
+Warren Weaver was an American scientist who designed several methods for developing Machine translation in 1949.
+
```

Comment:
It's always good to name hard facts.

For example, the institutions where he worked.